### ISSUE TITLE
Loading (StaticQuery)

### ISSUE THREAD

**Issue by morgvanny** [EmaSuriano/gatsby-starter-mate#2]
I'm not sure what I'm doing wrong, but if I so much as remove a Social Link from the About content type in Contentful, it seems to irrevocably break the build, even if I try to link the existing entry back in to the Social Links.

I get the following during `gatsby develop`:

GraphQL Error Unknown field `profile` on type `ContentfulAbout`

  file: src/components/Helmet.js

   1 |
   2 |       query HelmetQuery {
   3 |         contentfulAbout {
   4 |           name
   5 |           description
>  6 |           profile {
     |           ^
   7 |             favicon16: resize(width: 16) {
   8 |               src
   9 |             }
  10 |             favicon32: resize(width: 32) {
  11 |               src
  12 |             }
  13 |             bigIcon: resize(width: 192) {
  14 |               src
  15 |             }
  16 |             appleIcon: resize(width: 180) {

Then it still says Compiled successfully, but in the browser, anywhere where there should be any content at all from Contentful, it simply shows 'Loading (StaticQuery)'

I'm new to using both GraphQL and Contentful, so it could be entirely my fault, but it seems like simply clicking the 'x' on one social link (out of 4) shouldn't ruin everything and be unfixable without re-running `yarn setup` again.

It appears I can get away with archiving social links and nothing breaks, they just don't show up on the page. But why mustn't I remove any?

**Comment by morgvanny** [comment]
Ok, I've discovered if I delete the .cache directory and run it again, it works as I'd expect. So I'm guessing this is a gatsby problem, not specific to this starter. Please let me know if you think an update to any package may fix this though!

**Comment by EmaSuriano** [comment]
Hi @morgvanny, thank you for trying the starter! But remember it's not ready yet due to this annoying problem. 
I had the same issue, and I used the following solution: when I run `develop` or `build` command I delete the `.cache` folder, this is the command `"develop": "cleandir ./.cache && gatsby develop",`.

If you asked me is the worst, but it's the only workaround I found by now. Hope it helps you!
